modules/asset-manager-api/src/main/java/org/opencastproject/assetmanager/api/AssetManager.java

```diff
@@ -96,6 +96,24 @@
    */
   boolean setProperty(Property property);
 
+  /**
+   * Delete all properties for a given media package identifier
+   *
+   * @param mediaPackageId
+   *          Media package identifier
+   */
+  void deleteProperties(String mediaPackageId);
```

Comment:
I would be tempted to rename this delete**All**Properties so that it's clearer.

Reply:
But this does not delete all properties. It only deletes those who fit the given criteria which is the media package identifier here and the media package identifier and the namespace for the other method.

To me `assetManager.deleteAllProperties(…)` sounds like it's dropping all properties the asset manager has stored.